docs/docs/user/what-is-kpops.md

```diff
@@ -1,7 +1,22 @@
 # What is KPOps?
 
-KPOps is a CLI tool that allows deploying and destroying consecutive Kafka Streams applications, Producers, and Kafka 
-Connectors using an easy-to-read and write pipeline definition. KPOps comes with various abstractions that can simplify
-configuring multiple pipelines and steps within pipelines by sharing configuration between different applications,
-like producers or streaming applications. In addition, Kafka apps and its resources can be reset or cleaned up 
-again with a single command.
+With a couple easy commands in the shell and a [`pipeline.yaml`](#example) of under 30 lines, [KPOps](/) can [`destroy`](/user/references/cli-commands/#deploy), [`reset`](/user/references/cli-commands/#reset), [`clean`](/user/references/cli-commands/#clean) or [`deploy`](/user/references/cli-commands/#deploy) a pipeline[^1] to a Kubernetes cluster!
```

Comment:
```suggestion
With a couple easy commands in the shell and a [`pipeline.yaml`](#example) of under 30 lines, [KPOps](/) can [`deploy`](/user/references/cli-commands/#deploy), [`reset`](/user/references/cli-commands/#reset), [`clean`](/user/references/cli-commands/#clean) or [`destroy`](/user/references/cli-commands/#destroy) a Kafka pipeline[^1] to a Kubernetes cluster!
```

Reply:
I had it like that initially, but I think it doesn't sound good: "...destroy a Kafka pipeline to a Kubernetes cluster!". It seems I forgot to switch out the links tho..

What do you think of it now? I reworded it a bit